projects/components/src/list-view/list-view.component.scss

```diff
@@ -57,6 +57,30 @@ $horizontal-offset: 12px;
     }
   }
 
+  .metadata-row {
+    @include body-small();
+    display: flex;
+    padding: 0 20px;
+    gap: 8px;
+    align-items: center;
+
+    .marker {
+      width: 4px;
+      height: 4px;
+      border-radius: 50%;
+      background: $gray-5;
+    }
+    .key {
+      color: $gray-5;
+      font-weight: 400;
```

Comment:
nit: can we use an existing mixin (if any)?

Reply:
we usually override single properties like this. there is no mixin for font-weights